internal/lambgofile/lambgofile.go

```diff
@@ -62,6 +68,8 @@ type Config struct {
 
 	OutDirectory   string   `yaml:"outDirectory"`
 	ZippedFileName string   `yaml:"zippedFileName"`
+	RawExtraFlags  string   `yaml:"extraFlags"`
```

Comment:
Alternatively, if we don't want to add the `shlex` dependency, we could do the approach which lots of Go tools use, and just accept an array. I believe Docker does this. And something sorta closer to this project is GoReleaser. Here's their [config file](https://goreleaser.com/customization/build/), where they just have a `flags` array that is passed directly to `go build`.